go-controller/.gitignore

```diff
@@ -1,2 +1,3 @@
 _output
 _artifacts
+*.test
```

Comment:
Sorry to be picky, but are `*.test` files generated from running `make test`?

Reply:
If what you want is to exclude any artifact that is generated from running manual commands, then I suggest you add them to`.git/info/exclude`. It does the trick without polluting the `.gitignore` file with custom ignores.

Reply:
yeah I happen to have lots of these files now to catch unit test problems (built with go test -c), and I thought these files should never be commited anyway, so added them here

Reply:
I figured as much. As far as I understand, the `make test` target should not leave any artifact because it runs on a container. My advice is to add it to your `exclude` file instead as it is specific to how tests are run manually and not something always generated by a make target. But up to the team's decision, just my 0.02€ 😄